features/support/files_helpers.rb

```diff
@@ -23,6 +23,32 @@ def files_in_branches
 end
 
 
+def create_uncommitted_file options = {}
+  options.reverse_merge!(default_uncommitted_file_attributes)
+  IO.write options[:file_name], options[:file_content]
+  options[:file_name]
+end
+
+
+def default_uncommitted_file_attributes
+  {
+    file_name: "default_uncommitted_file_name_#{SecureRandom.urlsafe_base64}",
```

Comment:
Yeah that works too. I had just included it to be consistent with the default commit attributes, and it also made it clear which tests had been relying on the default file name of "uncommitted_file"